api/include/mfxstructures.h

```diff
@@ -470,6 +470,9 @@ enum {
     MFX_PROFILE_HEVC_MAIN10           =2,
     MFX_PROFILE_HEVC_MAINSP           =3,
     MFX_PROFILE_HEVC_REXT             =4,
+#if (MFX_VERSION >= 1031)
```

Comment:
Please replace 1031 here and in in other parts of this PR to MFX_VERSION_NEXT. There are CI configuration named **buildbot/build-mediasdk-api-next** which enable these features. API 1.31 scope already finalized.

Reply:
Please add respective changes in [mediasdk-man.md](./doc/mediasdk-man.md) to this PR.

Reply:
> Please replace 1031 here and in in other parts of this PR to MFX_VERSION_NEXT. There are CI configuration named **buildbot/build-mediasdk-api-next** which enable these features. API 1.31 scope already finalized.

In fact, I try to use MFX_VERSION_NEXT at first. But it seems not to work. The macro section isn't enabled. May I set it in CMake command line? I can double check it.

Reply:
You can pass `-DAPI:STRING=latest` to CMake. API version would become **1.31+** instead of just 1.31. Respective line in product config: https://github.com/Intel-Media-SDK/product-configs/blob/master/conf_linux_public.py#L149

Reply:
I tried it. It works fine. I will use MFX_VERSION_NEXT instead of hard code. Thanks.